neuralmonkey/encoders/factored_encoder.py

```diff
@@ -176,12 +171,12 @@ def _create_encoder_graph(self):
         self.attention_tensor = tf.nn.dropout(self.attention_tensor,
                                               self.dropout_placeholder)
 
-
     def feed_dict(self, dataset, train=False):
-        factors = {data_id: dataset.get_series(data_id) for data_id in self.data_ids}
+        factors = {data_id: dataset.get_series(
+            data_id) for data_id in self.data_ids}
```

Comment:
Já taky, za tohole může autotpep8, to pouštim na soubory, kde pekelně moc dlouhých řádek.